daemon/state.go

```diff
@@ -2,26 +2,16 @@ package daemon
 
 import (
 	"fmt"
-	"sync"
 	"time"
 
 	"github.com/docker/docker/daemon/execdriver"
 	"github.com/docker/docker/pkg/units"
+	"github.com/docker/docker/runconfig"
 )
 
 type State struct {
-	sync.Mutex
-	Running           bool
```

Comment:
these statuses should remain here IMHO, to me they don't have nothing to do with runconfig, wdyt?